webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -293,6 +316,36 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isLoadingSettings]);
 
+  useEffect(() => {
```

Comment:
nit: this whole useEffect could use more commenting

Reply:
will probably refactor this pattern to be clearer -- it's the same setup as the formset effect, but it's very verbose and we're doing a weird thing with the cleanup reference in order to get around `subscribe` only being called when the observable changes.

Reply:
Yeah, it's confusing in the other spot too, especially the cleanup

Reply:
@EmilyBonar refactored the effects -- is this easier to understand?

Reply:
Yeah that's a lot better, thanks!